presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanNode.java

```diff
@@ -60,7 +60,8 @@
         @JsonSubTypes.Type(value = ExplainAnalyzeNode.class, name = "explainAnalyze"),
         @JsonSubTypes.Type(value = ApplyNode.class, name = "apply"),
         @JsonSubTypes.Type(value = AssignUniqueId.class, name = "assignUniqueId"),
-        @JsonSubTypes.Type(value = LateralJoinNode.class, name = "lateralJoin")})
+        @JsonSubTypes.Type(value = LateralJoinNode.class, name = "lateralJoin"),
+        @JsonSubTypes.Type(value = AnalyzeFinishNode.class, name = "analyzecommit")})
```

Comment:
update name

Reply:
Agreed, the existing ones are a mess. I would go with `analyzeCommit` as the camel case makes it slightly more readable.